tf2_py/src/tf2_py.cpp

```diff
@@ -45,30 +45,20 @@
   } \
   } while (0)
 
-static PyObject *pModulerospy = NULL;
+static PyObject *pModulerclpy = NULL;
+static PyObject *pModulerclpytime = NULL;
+static PyObject *pModulebuiltininterfacesmsgs = NULL;
```

Comment:
Since this is a `.cpp` file, is there any reason not to use `nullptr` instead of `NULL` here?